### PR TITLE
clean up println's

### PR DESCRIPTION
**Pull request by cfregly**

### CODE REVIEW

File: docs/ml-guide.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ val training = sc.parallelize(Seq(
 // Create a LogisticRegression instance.  This instance is an Estimator.
 val lr = new LogisticRegression()
 // Print out the parameters, documentation, and any default values.
-println("LogisticRegression parameters:\n" + lr.explainParams() + "\n")
+println(s"LogisticRegression parameters:\n ${lr.explainParams()}")
```
Comment (mengxr) [inline docs/ml-guide.md]: Remove space after `\n`. The last `\n` should be preserved to create an empty line after this block.

**Comment by mengxr** [comment]
LGTM except a minor inline comment.

**Comment by srowen** [comment]
Also, this was opened against `branch-1.3`, can you just open against `master`?

**Comment by srowen** [comment]
Mind closing this PR? (Jenkins invocation) Obviously it's fine but needs to go in vs `master`.

**Pull request closed without merging**